### PR TITLE
(Centaur Port) Decode responses

### PR DESCRIPTION
**Pull request by cjllanwarne**
Porting the decoding in https://github.com/broadinstitute/centaur/pull/181 into the Cromwell client.

### CODE REVIEW

**Comment by Horneth** [comment]
👍

File: cromwellApiClient/src/main/scala/cromwell/api/CromwellClient.scala [hunk 1]
```diff
@@ -100,18 +103,31 @@ class CromwellClient(val cromwellUrl: URL, val apiVersion: String)(implicit acto
 
     options map (o => addToken(o.parseJson.asJsObject.convertTo[Map[String, JsValue]]).toJson.toString)
   }
+
+  private def decodeResponse(response: HttpResponse): HttpResponse = {
+    val decoder = response.encoding match {
+      case HttpEncodings.gzip ⇒
```
Comment (mcovarr) [inline cromwellApiClient/src/main/scala/cromwell/api/CromwellClient.scala]: unicode rockets?

**Comment by mcovarr** [comment]
:+1:

File: cromwellApiClient/src/main/scala/cromwell/api/CromwellClient.scala (hunk 1 above)
Comment (Horneth) [inline cromwellApiClient/src/main/scala/cromwell/api/CromwellClient.scala]: yeah that's my fault I copy pasted this from akka docs -_-

**Pull request merged**